lib/ra10ke.rb

```diff
@@ -53,12 +53,17 @@ def initialize(*args)
               when 'sha'
                 latest_ref = remote_refs['head'][:sha]
               when 'tag'
-                # we have to be opinionated here, due to multiple conventions only one can reign suppreme
-                # as such tag v#.#.# is what will pick.
+                # we have to be opinionated here, due to multiple conventions only the two main will be accepted
+                # v#.#.# or #.#.# is what we will pick.
+                tags = remote_refs['tags']
                 if ref.match(/^[vV]\d/)
-                  tags = remote_refs['tags']
                   version_tags = tags.select { |f| /^[vV]\d/.match(f) }
                   latest_ref = version_tags.keys.sort.last
+                elsif ref.match(/^\d[\.\d]*/)
```

Comment:
It allows to filter tags according to the convention found if the Puppetfile ref.
If for some historical reason a repo has the both naming convention, this block prevent from getting weird sort result.
It's not necessary, tell me.